src/main/java/org/hobbit/core/rabbit/CustomConsumer.java

```diff
@@ -0,0 +1,38 @@
+package org.hobbit.core.rabbit;
+
+import java.io.IOException;
+import java.util.concurrent.LinkedBlockingQueue;
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.DefaultConsumer;
+import com.rabbitmq.client.Delivery;
+import com.rabbitmq.client.Envelope;
+
+public class CustomConsumer extends DefaultConsumer {
```

Comment:
Could you please name it `QueingConsumer` or similar? This would reflect its functionality in a better way.

Apart from that, this class needs to be documented using Javadoc comments!